<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>611</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>601</width>
       <height>311</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Gestion des employés</string>
      </attribute>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>281</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Ajouter employé</string>
        </attribute>
        <widget class="QLineEdit" name="lineEditCin">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>50</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>50</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>           Cin</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>        Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>    Prénom</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_nom">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>90</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_prenom">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>130</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>15</y>
           <width>221</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Veuillez remplir les informations suivantes</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>170</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_age">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>170</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>50</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Tel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>100</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>150</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Departement</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_tel">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>50</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_adresse">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>100</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_depart">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>150</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_ajouter">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>210</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Afficher la liste des employés</string>
        </attribute>
        <widget class="QListView" name="listView">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>51</y>
           <width>561</width>
           <height>201</height>
          </rect>
         </property>
        </widget>
        <widget class="QScrollBar" name="verticalScrollBar">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>50</y>
           <width>16</width>
           <height>201</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>19</x>
           <y>10</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Trier</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>10</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>10</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Recherche</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Modifier employé</string>
        </attribute>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Cin</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>90</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>130</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>170</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>130</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>170</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>50</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Tel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>100</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>150</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Département</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>50</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>100</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_8">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>150</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>449</x>
           <y>210</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Supprimer employé</string>
        </attribute>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>90</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Cin</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_9">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>90</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>160</y>
           <width>80</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>gestion des département</string>
      </attribute>
      <widget class="QTabWidget" name="tabWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>281</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string>Ajouter département</string>
        </attribute>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>90</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>140</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Chef</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>190</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>NumEmp</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_id">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>40</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_nom_2">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>90</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_chef">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>140</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_numemp">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>190</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>110</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_8">
        <attribute name="title">
         <string>Afficher la liste des départements</string>
        </attribute>
        <widget class="QListView" name="listView_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>571</width>
           <height>192</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Trier</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_10">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>10</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_7">
         <property name="geometry">
          <rect>
           <x>439</x>
           <y>10</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>recherche</string>
         </property>
        </widget>
        <widget class="QScrollBar" name="verticalScrollBar_2">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>50</y>
           <width>16</width>
           <height>191</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_9">
        <attribute name="title">
         <string>Modifier département</string>
        </attribute>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>90</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_23">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>150</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Chef</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_24">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>220</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Numemp</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_11">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_12">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>90</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_13">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>150</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_14">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>210</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>110</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_10">
        <attribute name="title">
         <string>Supprimer département</string>
        </attribute>
        <widget class="QLabel" name="label_25">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>100</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_15">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>100</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_9">
         <property name="geometry">
          <rect>
           <x>209</x>
           <y>150</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
